src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/BaseCodeActionService.cs

```diff
@@ -16,22 +17,25 @@
 using OmniSharp.Mef;
 using OmniSharp.Models;
 using OmniSharp.Models.V2.CodeActions;
+using OmniSharp.Roslyn.CSharp.Helpers;
 using OmniSharp.Roslyn.CSharp.Services.Diagnostics;
 using OmniSharp.Roslyn.CSharp.Workers.Diagnostics;
 using OmniSharp.Roslyn.Utilities;
 using OmniSharp.Services;
 using OmniSharp.Utilities;
 using FixAllScope = OmniSharp.Abstractions.Models.V1.FixAll.FixAllScope;
 
+#nullable enable
```

Comment:
How dare you nullable! (j/k!)

Reply:
I know, we put too much time into it to use!